codebase/config/sync/google_analytics.settings.yml

```diff
@@ -0,0 +1,38 @@
+account: G-8435EBBK3D
```

Comment:
You might want to see how this could be an environment variable. That way the local dev environment wouldn't be using the same code as production. Another option would we to set up GA so that it only records activity that happens on the prod domain.

Reply:
Good point I didn't think of that